### PR TITLE
registry/ontologies.yaml -> ontologies.yml

### PR DESCRIPTION
**Pull request by nlharris**
This seemed to me to be the only obviously actionable issue left in #302. (registry/ontologies.rdf is still missing but I'm not sure how to fix that.)

### CODE REVIEW

File: README-sitedev.md [hunk 1]
```diff
@@ -77,9 +77,9 @@ jena is at https://archive.apache.org/dist/jena/binaries/apache-jena-3.10.0.tar.
 
 The dependencies should be visible in the [Makefile](Makefile). The basic idea is:
 
- * [ontology/](ontology/)*.md  --[extract yaml]--> [registry/ontologies.yaml](registry/ontologies.yaml) --> [_config.yml](_config.yml)
+ * [ontology/](ontology/)*.md  --[extract yaml]--> [registry/ontologies.yml](registry/ontologies.yml) --> [_config.yml](_config.yml)
 
-[registry/ontologies.yaml](registry/ontologies.yaml) is also used to
+[registry/ontologies.yml](registry/ontologies.yml) is also used to
 create [registry/ontologies.rdf](registry/ontologies.rdf) via a
```
Comment (cthoyt) [inline README-sitedev.md:83]: there are several RDF files already available:

1. ontologies.nt  (in the N-tuples format)
2. ontologies.ttl (in the turtle format)
3. ontologies.jsonld (in the JSON-LD format)

Even though content may be RDF, `*.rdf` typically isn't used as a file extension. You can list these three as an alternative to finish this up

Comment (nlharris) [reply README-sitedev.md]: Thanks, Charlie! See if my edit is what you intended.

BTW, I noticed that this documentation refers to the "prototype" obofoundry web page -- aren't we past the "prototype" phase at this point?

**Review by cthoyt — APPROVED** [review]

File: README-sitedev.md (hunk 1 above)
Comment (cthoyt) [reply README-sitedev.md]: you're welcome!

**Review by matentzn — APPROVED** [review]

File: README-sitedev.md (hunk 1 above)
Comment (matentzn) [reply README-sitedev.md]: > not sure if I can comment on the prototype status of OBO Foundry

I hope you said that @cthoyt with a slight smile and intellectual curiosity, not with hidden undertone :P

**Pull request merged**